MigrationGuide.md

```diff
@@ -0,0 +1,335 @@
+# Guide for migrating to OpenAI 2.0.0-beta.1 from OpenAI 1.11.0
+
+This guide is intended to assist in the migration to the official OpenAI library (2.0.0-beta.1) from [OpenAI 1.11.0][openai_1110], focusing on side-by-side comparisons for similar operations between libraries. In this document, these are often referred to as the 'new' and the 'old' OpenAI libraries, respectively.
+
```

Comment:
I'd recommend an explicit mention that, prior to 2.0, the `OpenAI` package was a community library not officially by OpenAI and that it's now the official library. Linking the recent changelog update could likely facilitate that, too!